src/screens/TermsConditionsScreen.tsx

```diff
@@ -28,17 +28,30 @@ interface Props {
   language: string;
 }
 
-export class TermsConditionsScreen extends React.PureComponent<Props> {
+interface State {
+  showWarring: boolean;
+  height: number;
+  agreedToTermsAndConditions: boolean;
+  agreedToPrivacyPolicy: boolean;
+}
+
+const TERMS_AND_CONDITIONS_MARGIN_BOTTOM = 32;
+export class TermsConditionsScreen extends React.PureComponent<Props, State> {
   state = {
-    canGo: false,
     showWarring: false,
     height: 500,
+    agreedToTermsAndConditions: false,
+    agreedToPrivacyPolicy: false,
   };
 
-  handleCanGo = () => {
-    this.setState({
-      canGo: true,
-    });
+  toggleAgreementToTermsAndConditions = () => {
+    const { agreedToTermsAndConditions } = this.state;
```

Comment:
I know it's strongly subjective, but I don't see a point of destructing `this.state` here. For me, it makes the code harder to read

Reply:
I'll keep it, I like destructuryzation :D